### DISCUSSION TITLE
How to delete one month old record from a supabase public table automatically?

### DISCUSSION THREAD

**Discussion by siamahnaf** [supabase/supabase#28864] [Questions]
Hello, I create a supabase public table. Suppose that table name is "emails". So here now I want to delete all record which one is already  1 month old automatically. I mean if one record fill one month from it's creation date it should delete from table automatically. I am not getting any idea. If I need to run any schedule job or cron job, then how can I do it, I need details.  I am using free version of supabase currently.

**Comment by GaryAustin1** [comment]
You would use a cron task running every hour or day (some time period you are happy with).
https://www.citusdata.com/blog/2016/09/09/pgcron-run-periodic-jobs-in-postgres/
This is all Postgres so there are probably many more examples or any AI should generate good code for it.
